src/Settings/Settings.php

```diff
@@ -23,31 +23,12 @@ final class Settings implements SettingsInterface
 {
     public const DEFAULT_KEY = 'default';
 
-    /**
-     * @var Metadata
-     */
     private Metadata $metadata;
 
-    /**
-     * @var SettingRepositoryInterface
-     */
     private SettingRepositoryInterface $settingRepository;
 
-    /**
-     * @var array|null
-     */
-    private ?array $settingsByChannelAndLocale;
-
-    /**
-     * @var array|null
-     */
-    private ?array $settingsByChannelAndLocaleWithDefault;
-
```

Comment:
Why it is removed ?

Reply:
mistake ... with the refactoring of `getSettingsByChannelAndLocale` method, PHPStorm tells me that the variables are not used